apollo-router-core/src/context.rs

```diff
@@ -45,55 +45,101 @@ impl From<Context<http_compat::Request<Request>>> for Context<Arc<http_compat::R
 }
 
 impl<T> Context<T> {
-    pub fn extensions(&self) -> Arc<RwLock<Object>> {
-        self.extensions.clone()
+    pub fn get<K: Into<String>, V: for<'de> serde::Deserialize<'de>>(
```

Comment:
NIT: Could we have the trait bounds in a where clause ? I think it will be more readable

Reply:
Done